### DISCUSSION TITLE
Allow autograder to report a 0-max-score test as failing

### DISCUSSION THREAD

**Discussion by mister-walter** [gradescope/autograder_samples#67] [Ideas]
Currently, it doesn't appear that there is any way to get the Gradescope UI to report a test with `max_score`=0 as failing, regardless of the reported score of the test. This could be used to report errors to the student that might not be easily associated with a particular part of a rubric.

A concrete use case: notifying students that their helper functions do not have full test coverage, where the point deduction should be calculated for the problem that the helper function is used for. It may be nontrivial to identify which problem the helper function is associated with, so in this case it might be best to simply tell the student that helper function X does not have full test coverage, and that they may lose points when a manual grader reviews their work.

**Comment by liamsaliba** [comment]
I got around this by having the max score as 1e-10; so 0/1e-10 is fail and 1e-10/1e-10 is pass.

Doesn't look the greatest in the Gradescope UI though.

**Comment by ibrahima** [comment]
Hi @mister-walter. Thanks for the feature request! We've just released this capability. Please see our [autograder documentation for how to use it!](https://gradescope-autograders.readthedocs.io/en/latest/specs/#test-case-status)
